src/services/utils.ts

```diff
@@ -68,7 +68,11 @@ export function filterTextItems(
         const useItem = shouldProcessItem(item, vars, options);
 
         if (useItem) {
-            result.push(item.text);
+            if (Array.isArray(item.text)) {
+                result.push(...item.text);
+            } else {
+                result.push(item.text);
+            }
```

Comment:
I suppose we can shorten it
```suggestion
            result.push(Array.isArray(item.text)  ? (...item.text) : item.text)
```

Reply:
Typescript don't understand this syntax, so, maybe better leave it as is